test_runner/fixtures/pg_version.py

```diff
@@ -52,7 +52,7 @@ def _missing_(cls, value) -> Optional["PgVersion"]:
         return None
 
 
-DEFAULT_VERSION: PgVersion = PgVersion.V14
+DEFAULT_VERSION: PgVersion = PgVersion.V15
```

Comment:
This is from your original diff, please confirm it has no effect on anything? Or should it be changed in a separate PR if it has an effect?

Reply:
That's ok to keep it here, we don't use `DEFAULT_VERSION` much.